frontend/javascripts/navbar.tsx

```diff
@@ -353,7 +353,7 @@ function HelpSubMenu({
       {isAuthenticated ? (
         <>
           <Menu.Item key="get_help" onClick={() => setIsHelpModalOpen(true)}>
-            Get help / Ask a question
+            Ask a question
```

Comment:
the rest of the menu items are in title case, so I'd adapt this here for consistency.

Reply:
Done